lib-lua/themes/nominatim/presets.lua

```diff
@@ -172,6 +172,10 @@ module.MAIN_TAGS_POIS = function (group)
                  no = group,
                  yes = group,
                  fire_hydrant = group},
+    entrance = {'always',
+                no = group},
+    ["routing:entrance"] = {'always',
```

Comment:
Lets do a little trick here and set this instead of 'always' to 

    exclude_when_key_present('entrance')

This will avoid that two rows are created when entrance and routing:entrance tags are both present in the same OSM object.